### DISCUSSION TITLE
How to force a package (bullmq) to be bundled into bundledDependencies rather than inlined

### DISCUSSION THREAD

**Discussion by Dodobibi** [nuxt/nuxt#21455] [Questions]
I use 'bullmq' on server side.

Because bullmq need to access filesystem of it's package, it cannot be inlined in server.mjs.

If I set it in external, it's not inlined, but not exported in node_modules or bundledDependencies.

Is there a way (a nitro or nuxt configuration) to force a package to be in bundledDependencies rather than inlined ?

Thank's for your help !

**Comment by IqroNegoro** [comment]
has you found yet?
